### PR TITLE
[ru] Localize `Ingress`

### LINKED ISSUE #3122: [ru] Localize `Ingress`
### Description

This Issue is to localize [`Ingress`](https://github.com/cncf/glossary/blob/main/content/en/ingress.md) into Russian.

### Checklist before opening your PR (put `x` in the checkboxes) — note that you have to _add this checklist to your PR and check all the boxes for it to be approved_.
- [] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.
    
**ETA: TBD**

Useful links:
[Как внести свой вклад](https://glossary.cncf.io/ru/contribute/)
[Руководство по стилю](https://glossary.cncf.io/ru/style-guide/)

[Slack channel](https://app.slack.com/client/T08PSQ7BQ/C05G46RMQTX)
[Telegram channel](https://t.me/ru_cloudnative)

### PR DESCRIPTION
**Pull request by kirkonru**
### Describe your changes
Localize `Ingress`

### Related issue number or link (ex: `resolves #issue-number`)
resolves #3122 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR uses the **dev-ru** branch as a source/target.
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/ru/ingress.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+title: Ingress
+status: Completed
+category: technology
+tags: ["fundamental"]
+---
+
+Ingress — набор правил, которые помогают управлять интернет-трафиком, поступающим извне в контейнер или группу контейнеров, работающих в кластере.
+Он состоит из двух элементов: ресурса Ingress и Ingress-контроллера.
+Ресурс Ingress — это конфигурационный файл, который живет вместе с другими файлами манифестов и позволяет администраторам настраивать маршрутизацию внешнего трафика.
+Ingress-контроллер — серверная технология, которая маршрутизирует трафик в соответствии с конфигурацией в Ingress-ресурсе.
```
Comment (shurup) [inline content/ru/ingress.md:11]: Mentioning a «web server» ( not just a «server») is essential.
```suggestion
Ingress-контроллер — технология веб-сервера, которая маршрутизирует трафик в соответствии с конфигурацией в Ingress-ресурсе.
```

File: content/ru/ingress.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+## Какую проблему решает
+
+Нативные облачные приложения состоят из множества сервисов. Часто эти [сервисы](/service/) должны быть доступны через интернет, чтобы пользователи могли обратиться к ним через браузер.
```
Comment (shurup) [inline content/ru/ingress.md:15]:
```suggestion
Нативные облачные веб-приложения состоят из множества сервисов. Часто эти [сервисы](/service/) должны быть доступны через интернет, чтобы пользователи могли обратиться к ним через браузер.
```

File: content/ru/ingress.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+В [Kubernetes](/ru/kubernetes/), чтобы сделать эти сервисы доступными для потребителей, необходимо открыть их для внешнего мира.
```
Comment (shurup) [inline content/ru/ingress.md:16]:
```suggestion
Чтобы сервисы приложения, запущенного в [Kubernetes](/ru/kubernetes/), стали доступными для внешнего мира, необходимо выполнить специальные действия.
```

File: content/ru/ingress.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+Проще всего было бы использовать балансировщик нагрузки Kubernetes.
+Но создание такого сервиса означает появление нового компонента в базовой инфраструктуре.
+Возникают дополнительные затраты, расходуются ресурсы на управление. Кроме того, у каждого вновь созданного балансировщика нагрузки будет свой собственный, уникальный внешний IP-адрес.
```
Comment (shurup) [inline content/ru/ingress.md:19]:
```suggestion
Возникнут дополнительные затраты и накладные расходы на управление. Кроме того, у каждого вновь созданного балансировщика нагрузки будет свой собственный внешний IP-адрес.
```

File: content/ru/ingress.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+В результате пострадает пользовательский опыт, ведь мы, как пользователи, не готовы каждый раз вводить новый URL-адрес для доступа к приложению.
```
Comment (shurup) [inline content/ru/ingress.md:20]:
```suggestion
Из-за этого пострадают и сами пользователи, поскольку придётся каждый раз вводить новый URL-адрес для доступа к приложению.
```

File: content/ru/ingress.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+## Как именно решает проблему
+
+Ресурс Ingress позволяет настроить балансировку и маршрутизацию трафика к сервисам приложения.
+Ingress-контроллер открывает единую точку входа через URL (www.example-url.com) и выполняет фактическую маршрутизацию и балансировку на основе различных URL-путей (www.example-url.com/path).
```
Comment (shurup) [inline content/ru/ingress.md:25]:
```suggestion
Ingress-контроллер открывает единую точку входа через URL (www.example-url.com) и выполняет фактическую маршрутизацию и балансировку, руководствуясь различными URL-путями (www.example-url.com/path).
```

File: content/ru/ingress.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+Ingress-контроллер работает в кластере и интерпретирует правила, определенные в ресурсе Ingress.
+Выбор конкретного Ingress-контроллера из набора возможных технологий (Nginx, Traefik, HAProxy и т. д.), зависит от операторов кластера, в котором работает веб-приложение.
```
Comment (shurup) [inline content/ru/ingress.md:27]:
```suggestion
Выбор конкретного Ingress-контроллера из существующих технологий (Nginx, Traefik, HAProxy и т. д.) делают сами администраторы кластера, в котором работает веб-приложение.
```

**Review by shurup — COMMENTED** [review]
Another bunch of opinionated, small corrections from me — feel free to agree with them or not :wink:

**Review by shurup — APPROVED** [review]
Thanks, LGTM!

**Review by tym83 — APPROVED** [review]

**Pull request merged**